java/src/org/openqa/selenium/remote/http/jdk/JdkHttpClient.java

```diff
@@ -51,22 +51,12 @@
 import org.openqa.selenium.TimeoutException;
 import org.openqa.selenium.UsernameAndPassword;
 import org.openqa.selenium.WebDriverException;
-import org.openqa.selenium.remote.http.BinaryMessage;
-import org.openqa.selenium.remote.http.ClientConfig;
-import org.openqa.selenium.remote.http.CloseMessage;
-import org.openqa.selenium.remote.http.ConnectionFailedException;
-import org.openqa.selenium.remote.http.HttpClient;
-import org.openqa.selenium.remote.http.HttpClientName;
-import org.openqa.selenium.remote.http.HttpMethod;
-import org.openqa.selenium.remote.http.HttpRequest;
-import org.openqa.selenium.remote.http.HttpResponse;
-import org.openqa.selenium.remote.http.Message;
-import org.openqa.selenium.remote.http.TextMessage;
-import org.openqa.selenium.remote.http.WebSocket;
+ import org.openqa.selenium.remote.http.*;
```

Comment:
We try to have all imports declared individually.

Reply:
Sorry, intellij did refactor it on save. Should i restore the imports in a new PR?

Reply:
Please restore them and push directly to `trunk`. No need for a PR.